lib/datasource/npm/get.ts

```diff
@@ -217,13 +217,9 @@ export async function getDependency(
       };
       if (res.time?.[version]) {
         release.releaseTimestamp = res.time[version];
+        const relDate = DateTime.fromISO(release.releaseTimestamp);
         release.canBeUnpublished =
-          DateTime.local()
-            .startOf('day')
-            .diff(
-              DateTime.fromISO(release.releaseTimestamp).startOf('day'),
-              'days'
-            ).days === 0;
+          relDate.isValid && DateTime.local().diff(relDate, 'days').days < 1;
```

Comment:
Use less or equal?
```suggestion
          relDate.isValid && DateTime.local().diff(relDate, 'days').days <= 1;
```

Reply:
I think <1 to be consistent with previous logic